src/rewrite_generic.jl

```diff
@@ -55,14 +55,57 @@ function _is_kwarg(expr, kwarg::Symbol)
     return Meta.isexpr(expr, :kw) && expr.args[1] == kwarg
 end
 
+function _rewrite_elseif!(if_expr, expr::Any)
+    push!(if_expr.args, esc(expr))
+    return false
+end
```

Comment:
I need to add a test that catches this method. It think it's only needed if you don't build the expression via a macro.